src/nginx/t/config_rollouts_managed.t

```diff
@@ -154,6 +159,18 @@ is( $response_body, <<'EOF', 'Shelves returned in the response body.' );
 EOF
 
 
+# verify the first /endpoints_status
+$response = ApiManager::http_get($NginxPort, '/endpoints_status' );
+( $response_headers, $response_body ) = split /\r\n\r\n/, $response, 2;
+like( $response_headers, qr/HTTP\/1\.1 200 OK/, 'Returned HTTP 200.' );
+my $endpoints_status = decode_json( $response_body );
```

Comment:
Output of the /endpoints_status contains  system and process statuses data and statistics.
Most of data can be different from each request.
We only need to compare serviceConfigRollouts.